immuneML/reports/ml_reports/GeneratorReportPWM.py

```diff
@@ -0,0 +1,107 @@
+import plotly.graph_objects as go
+
+from pathlib import Path
+from numpy import array
+from immuneML.data_model.dataset.Dataset import Dataset
+from immuneML.ml_methods.UnsupervisedMLMethod import UnsupervisedMLMethod
+from immuneML.reports.ReportOutput import ReportOutput
+from immuneML.reports.ReportResult import ReportResult
+from immuneML.reports.ml_reports.UnsupervisedMLReport import UnsupervisedMLReport
+from immuneML.util.PathBuilder import PathBuilder
+import numpy as np
+import logomaker
+import matplotlib.pylab as plt
+plt.style.use("seaborn")
```

Comment:
I suggest not to set anything globally, but only within your function for plotting